<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>activity_cell</class>
 <widget class="QWidget" name="activity_cell">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="activity_name"/>
     </item>
     <item>
      <widget class="QComboBox" name="activity_type"/>
     </item>
     <item>
      <widget class="QPushButton" name="delete_activity_button">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
